CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SyntaxNodeFile.swift

```diff
@@ -0,0 +1,359 @@
+                      leftParen: .leftParenToken(),
+                      argumentList: TupleExprElementListSyntax {
+                        TupleExprElementSyntax(expression: ExprSyntax("(SyntaxArena(), (\(raw: parameters)))"))
+                      },
+                      rightParen: .rightParenToken(),
+                      trailingClosure: ClosureExprSyntax(
+                        signature: ClosureSignatureSyntax(
+                          input: ClosureSignatureSyntax.Input.input(
+                            ParameterClauseSyntax {
+                              FunctionParameterSyntax(firstName: .identifier("arena"))
+                              FunctionParameterSyntax(firstName: .wildcardToken())
+                            }
+                          )
+                        )
+                      ) {
+                        if !node.children.isEmpty {
+                          VariableDeclSyntax(letOrVarKeyword: .keyword(.let)) {
+                            PatternBindingSyntax(
+                              pattern: PatternSyntax("layout"),
+                              typeAnnotation: TypeAnnotationSyntax(type: TypeSyntax("[RawSyntax?]")),
+                              initializer: InitializerClauseSyntax(
+                                value: ArrayExprSyntax {
+                                  for child in node.children {
+                                    if child.isOptional {
+                                      ArrayElementSyntax(expression: ExprSyntax("\(raw: child.swiftName)?.raw"))
+                                    } else {
+                                      ArrayElementSyntax(expression: ExprSyntax("\(raw: child.swiftName).raw"))
+                                    }
+                                  }
+                                }
+                              )
+                            )
+                          }
+                        }
+
+                        if !node.children.isEmpty {
+                          DeclSyntax("""
+                            let raw = RawSyntax.makeLayout(
+                              kind: SyntaxKind.\(raw: node.swiftSyntaxKind), from: layout, arena: arena,
+                              leadingTrivia: leadingTrivia, trailingTrivia: trailingTrivia)
+                            """)
+                        } else {
+                          DeclSyntax("let raw = RawSyntax.makeEmptyLayout(kind: SyntaxKind.\(raw: node.swiftSyntaxKind), arena: arena)")
+                        }
+
+                        StmtSyntax("return SyntaxData.forRoot(raw)")
+                      }
+                    )
+                  )
+                )
+              }
+
+            ExprSyntax("self.init(data)")
+          }
+
+          if node.hasOptionalBaseTypeChild {
+            // TODO: Remove when we no longer support compiling in Swift 5.6. Change the
+            // above constructor to use `Optional<BaseType.none` instead.
```

Comment:
Nit, missing `>` after `none`.

Reply:
I think it should be after `BaseType` and before the `.`

Reply:
Oh, yes. I only noticed the missing `>` and didn’t actually read the line 😬